src/js/components/homepage/features/CovidFeature.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * CovidFeature.jsx
+ * Created By Jonathan Hill 04/23/2020
+ */
+
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+/* eslint-disable import/prefer-default-export */
```

Comment:
Could we still `export default` this component at the bottom of the file? This would also allow for consistent import statements / prevent having to look up whether it should be imported as `{ CovidFeature }` or `CovidFeature`